Datadog/Example/Debugging/DebugRUMSessionViewController.swift

```diff
@@ -54,13 +54,13 @@ private class DebugRUMSessionViewModel: ObservableObject {
                     self?.modifySessionItem(type: .view, label: key) { mutableSessionItem in
                         mutableSessionItem.isPending = false
                         mutableSessionItem.stopAction = nil
-                        Global.rum.stopView(key: key)
+                        RUMMonitor.shared().stopView(key: key)
```

Comment:
non blocking

I would prefer `RUMMonitor.shared()` singleton instance as a instance member of `DebugRUMSessionViewModel`

Reply:
By keeping this static access, we allow lazy init of the SDK. With a instance member we would need to make sure that the SDK is initialised before instantiating the `DebugRUMSessionViewModel`. So we can keep previous behaviour here 👍

Reply:
IMO it's fine to use singleton directly within the scope of the example app.